packages/schematics/angular/ng-new/schema.json

```diff
@@ -95,19 +96,33 @@
     "routing": {
       "type": "boolean",
       "description": "Generates a routing module.",
-      "default": false
+      "default": false,
+      "x-prompt": "Will the project use Angular routing?"
     },
     "prefix": {
       "type": "string",
       "format": "html-selector",
       "description": "The prefix to apply to generated selectors.",
+      "minLength": 1,
       "default": "app",
-      "alias": "p"
+      "alias": "p",
+      "x-prompt": "Choose an HTML selector prefix for the project:"
     },
     "style": {
       "description": "The file extension to be used for style files.",
       "type": "string",
-      "default": "css"
+      "default": "css",
+      "x-prompt": {
+        "message": "Select preferred style format for the project",
+        "type": "list",
+        "items": [
+          { "value": "css", "label": "CSS" },
```

Comment:
I think the Style options should be ordered by popularity, so:

1. css
2. scss
3. sass
4. less
5. stylus

Reply:
Done.